docs/konnectors-workflow.md

```diff
@@ -665,6 +665,20 @@ type with the following parameter:
 - `redirect_uri`, with an URL like `https://oauthcallback.mycozy.cloud/accounts/paypal/redirect`
 - `client_id`, with the client ID given by Budget Insight
 - `auth_endpoint`, with `https://{domain}.biapi.pro/2.0/webauth` (with the correct `domain`)
+
+### BI Weview
+
+A new integration of Budget Insight is available for all the proposed banks.
+This is a workflow similar to OAuth called
+[webview](https://docs.budget-insight.com/reference/webview)
+
+It is possible to use this workflow for konnectors by registering an account
+type with the following parameter:
+
+- `grant_mode`, with `bi_webview+secret` as the value
+- `redirect_uri`, with an URL like `https://oauthcallback.mycozy.cloud/accounts/paypal/redirect`
+- `client_id`, with the client ID given by Budget Insight
+- `auth_endpoint`, with `https://{domain}.biapi.pro/2.0/auth/webview/fr/connect` (with the correct `domain`)
```

Comment:
:point_down: Is the `reconnect_endpoint` for `bi_webauth`, `bi_webview`, or both?

Reply:
Both

Reply:
So, we should keep this line in `Webauth` section, right?

Reply:
Désolé je n'avais pas compris tout de suite. La ligne reconnect_endpoint est de nouveau dans la section webauth.